mmseg/ops/inverted_residual_module.py

```diff
@@ -0,0 +1,70 @@
+from mmcv.cnn import ConvModule, build_norm_layer
+from torch import nn
+
+
+class InvertedResidual(nn.Module):
+    """Inverted Residual Module
+    Args:
+        inp (int): input channels.
+        oup (int): output channels.
+        stride (int): downsampling factor.
+        expand_ratio (int): 1 or 2.
+        dilation (int): Dilated conv. Default: 1.
+        conv_cfg (dict | None): Config of conv layers. Default: None.
+        norm_cfg (dict | None): Config of norm layers. Default:
+            dict(type='BN').
+        act_cfg (dict): Config of activation layers. Default:
+            dict(type='ReLU6').
+    """
+
+    def __init__(self,
+                 inp,
+                 oup,
+                 stride,
+                 expand_ratio,
+                 dilation=1,
+                 conv_cfg=None,
+                 norm_cfg=dict(type='BN'),
+                 act_cfg=dict(type='ReLU6')):
```

Comment:
We may use similar variable name as [mmcls](https://github.com/open-mmlab/mmclassification/blob/master/mmcls/models/backbones/mobilenet_v2.py#L14).